### ISSUE TITLE
[Accessibility] Toggle control doesnt allow to specify aria-label and aria-description 

### ISSUE THREAD

**Issue by pankajkhanzode** [OfficeDev/office-ui-fabric-react#1092]
### Bug Report

- __Package version(s)__: (latest)
- __Browser and OS versions__: (all)

#### Describe the issue:

When there is no Label on Toggle control, there is still a need to specify ARIA label and description for screen reader to read and tell the context of toggle button.

#### Actual behavior:

There is no prop exposed that takes ARIA label not it adds anything specified in native props (like image).

#### Expected behavior:

Allow ARIA label and description to be specified.

### If applicable, please provide a codepen repro:

https://fabricreact.azurewebsites.net/fabric-react/master/#/examples/toggle

**Comment by yaananth** [comment]
dup of #1039
